File: w07/d01/Jane_Shin/multiples_monday.js
function multiples_monday(number) {
	var multiples_sum = 0 
	for (var i = 0; i < number; i++) {
		if (i%3 === 0 || i%5 === 0) {
			multiples_sum = multiples_sum + i;
		}
	};
	return multiples_sum
};

console.log(multiples_monday(1000));
iconsole.log(multiples_monday(30));

File: w07/d01/Jane_Shin/multiples_monday.rb
def multiples_monday(number)
	sum = 0
	i = 0
	while (i < number) do
		if (i%3 == 0 || i%5 == 0)
			sum += i
		end
		i += 1
	end
	puts sum
end

multiples_monday(1000)


# #====== keri's solution

# sum = 0

# (1...1000).each do |x|
# 	if (x%3 == 0 || x%5 == 0)
# 		sum += x
# 	end
# end

# puts sum
